File: 백준/Silver/1850. 최대공약수/최대공약수.py
import sys 
input = sys.stdin.readline
sys.setrecursionlimit(10**6)
def gcd(a, b):
	while a > 0 :
		a, b = b%a, a	
	return b

a, b = map(int, input().split())
res = gcd(a,b)
for _ in range(res):
	print(1, end='')

File: 백준/Silver/1850. 최대공약수/최대공약수.go
package main

import (
	"bufio"
	"fmt"
	"os"
)

func Gcd(a, b int) int {
	for a > 0 {
		a, b = b%a, a
	}
	return b
}

func main() {
	reader := bufio.NewReader(os.Stdin)
	writer := bufio.NewWriter(os.Stdout)
	defer writer.Flush()
	var a, b, res int
	fmt.Fscanf(reader, "%d %d\n", &a, &b)
	res = Gcd(a, b)
	for i := 0; i < res; i++ {
		fmt.Fprintf(writer, "%d", 1)
	}
}
